databricks/koalas/frame.py

```diff
@@ -163,8 +163,44 @@ def corr(self, method='pearson'):
         """
         return corr(self, method)
 
-    @derived_from(pd.DataFrame)
     def iteritems(self):
+        """
+        Iterator over (column name, Series) pairs.
+
+        Iterates over the DataFrame columns, returning a tuple with
+        the column name and the content as a Series.
+
+        Yields
```

Comment:
Let's make this API yield or fix the doc.

Reply:
I am fixing the doc for now.